cdap-docs/developers-manual/source/data-exploration/custom-datasets.rst

```diff
@@ -2,10 +2,10 @@
     :author: Cask Data, Inc.
     :copyright: Copyright © 2014-2015 Cask Data, Inc.
 
-.. _data-exploration:
+.. _custom-dataset-exploration:
 
 ============================================
-Data Exploration
+Custom Dataset Exploration
```

Comment:
I think the reason why I titled this the way I did and not "Custom Dataset" is that it covers both Custom and System Datasets that meet the two requirements. Is there any reason why not to keep the same logic? I also like to keep the sidebar titles and page titles as close in sync as possible, so you don't get surprises. One problem with this title is that it's a little ambiguous: is the "Custom" modifying the Dataset or the Exploration? I know what you mean, but readers might not.